ratis-examples/src/main/bin/stop-counter-server.sh

```diff
@@ -0,0 +1,18 @@
+ #!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ kill $(jps -lv | grep 'ratis' | grep 'CounterServer' | grep -v 'grep' | awk '{print $1}')
```

Comment:
Out of curiosity: will this way leave garbages generated by servers locally?

Reply:
In fact, I think `ratis-examples/src/main/bin./stop-all.sh` will server your stop server purpose.